deploy/examples/import-external-cluster.sh

```diff
@@ -197,11 +197,11 @@ parameters:
   dataPool: $RBD_METADATA_EC_POOL_NAME
   imageFormat: "2"
   imageFeatures: layering
-  csi.storage.k8s.io/provisioner-secret-name: $CSI_RBD_PROVISIONER_SECRET_NAME
+  csi.storage.k8s.io/provisioner-secret-name: rook-$CSI_RBD_PROVISIONER_SECRET_NAME
```

Comment:
csi.storage.k8s.io/provisioner-secret-name: "rook-""$CSI_RBD_PROVISIONER_SECRET_NAME"

This way would prevent word splitting.

Reply:
kubectl throws

 ```
error: error parsing STDIN: error converting YAML to JSON: yaml: line x: did not find expected key
``` 
Using ` "rook-""$CSI_RBD_PROVISIONER_SECRET_NAME" ` inside "Here Tag". Are you okay with `"rook-$CSI_RBD_PROVISIONER_SECRET_NAME"`

Reply:
What are you doing with `Kubectl` here with the import script?

Reply:
Try converting this simple yaml to json, then you will get my point
```yaml
foo : "one-""two"
```

Reply:
I understood your point about why it failed, but the ask is why it's even needed??

We can go ahead and directly run the shell script, right?

Reply:
Okay didn't realize we are creating a CR file there and running,

Fine with that go ahead:),
Thanks!!